### PR TITLE
French - add /miscl to match French Notion

### PR DESCRIPTION
**Pull request by hchiam**
another copy-paste like in #640, but i left the `---`s in since we'll need to do further work anyways to bring the French translation up to date with the latest English text anyways

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/current/miscl/trickery.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 2
+---
+
+# 🟢 \*\*\*\*Détection de tricherie
```
Comment (trigaten) [inline i18n/fr/docusaurus-plugin-content-docs/current/miscl/trickery.md:5]: `\*\*\*\*` ?

Comment (hchiam) [reply i18n/fr/docusaurus-plugin-content-docs/current/miscl/trickery.md:5]: whoops, sorry, totally a case of tunnel vision! good catch. i'll fix the other instances too

**Review by hchiam — COMMENTED** [review]
Ready for review.

File: i18n/fr/docusaurus-plugin-content-docs/current/miscl/detect.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Il fonctionne en calculant les probabilités logarithmiques du modèle censé avoir généré le texte et en les comparant à des altérations aléatoires du texte provenant d'un autre modèle de langage générique pré-entraîné. De cette manière, DetectGPT est capable d'identifier la probabilité que le passage ait été généré en utilisant uniquement les courbes de probabilité !
+
+## **Note**
+
+Pour une discussion supplémentaire sur le sujet des détecteurs et la façon dont les gens les trompent, voir [cet article](https://learnprompting.org/docs/miscl/trickery).
+
+---
+
+1. Bansal, A., yeh Ping-Chiang, Curry, M., Jain, R., Wigington, C., Manjunatha, V., Dickerson, J. P., & Goldstein, T. (2022). Certified Neural Network Watermarks with Randomized Smoothing. [↩](https://learnprompting.org/docs/miscl/detect#fnref-1)
```
Comment (trigaten) [inline i18n/fr/docusaurus-plugin-content-docs/current/miscl/detect.md:41]: Hmm look at how citations are done [here](https://learnprompting.org/contribute#prompt-engineering-techniques) and in other files.

Comment (hchiam) [reply i18n/fr/docusaurus-plugin-content-docs/current/miscl/detect.md:41]: nice! one less thing to worry about during translation. if i understand correctly, i don't need to edit bibliography.bib when it comes to translations

File: i18n/fr/docusaurus-plugin-content-docs/current/miscl/trickery.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Discussion
+
+L'un des domaines les plus controversés où ces types de techniques entrent en jeu est celui de l'éducation. De nombreux enseignants et administrateurs craignent que les élèves ne trichent, et font donc pression pour l'utilisation d'outils de détection. Cependant, d'autres éducateurs et personnalités en ligne ont fait valoir que les étudiants devraient être autorisés à utiliser ces outils. Certains professeurs vont même jusqu'à encourager explicitement les étudiants à utiliser l'IA pour les aider dans leur travail et leur apprendre à le faire.
+
+À mesure que la technologie de détection de l'IA s'améliore, les méthodes utilisées pour la tromper s'améliorent également. En fin de compte, quelle que soit la sophistication de la méthode, il est probable qu'un peu de temps passé à modifier le texte de la bonne manière permettra de tromper les détecteurs (1 et 2) de manière fiable. Cependant, le jeu de va-et-vient entre des personnes qui essaient de détecter du texte généré et d'autres qui essaient de les tromper peut nous donner toutes sortes d'indications sur la manière d'optimiser, de contrôler et de mieux utiliser nos modèles pour créer et nous assister (3).
+
+---
```
Comment (hchiam) [inline i18n/fr/docusaurus-plugin-content-docs/current/miscl/trickery.md]: citations will be auto-generated here too

**Review by hchiam — COMMENTED** [review]
Ready for review again. 

Citations at the bottom will be auto-generated - remove citations that someone copy-pasted into Notion.

File: i18n/fr/docusaurus-plugin-content-docs/current/miscl/detect.md (hunk 2 above)
Comment (trigaten) [reply i18n/fr/docusaurus-plugin-content-docs/current/miscl/detect.md]: Correct!

**Pull request merged**